src/Raven.Server/Documents/Handlers/RevisionsHandler.cs

```diff
@@ -180,6 +181,24 @@ public async Task GetRevisionsFor()
         public async Task Revert()
         {
             RevertRevisionsRequest configuration;
+            var collection = GetQueryOptionalStringValue("collection")?.Trim();
+
+            // Check if collection exists in the database
+            if (collection != null && CollectionExistsInDatabase(collection)==false)
```

Comment:
this check is invalid, because you might have revisions from a collection that does not exist anymore

Reply:
Removed